overlord/snapstate/snapmgr.go

```diff
@@ -117,6 +117,9 @@ type SnapState struct {
 	Aliases             map[string]*AliasTarget `json:"aliases,omitempty"`
 	AutoAliasesDisabled bool                    `json:"auto-aliases-disabled,omitempty"`
 	AliasesPending      bool                    `json:"aliases-pending,omitempty"`
+
+	// UserID of the user requesting the install
```

Comment:
the comment is not quite correct because this will also track the last user that refreshed the snap (at least with the current code as it stands)